domenetjenester/behandling/src/main/java/no/nav/foreldrepenger/behandling/aksjonspunkt/BekreftetAksjonspunktDto.java

```diff
@@ -12,7 +14,8 @@
 import no.nav.vedtak.util.InputValideringRegex;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
-@JsonTypeInfo(use = JsonTypeInfo.Id.NAME)
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY,
+    property = "@type")
```

Comment:
Hva er effekten av denne? Bør den ikke med på OverstyringAksjonspunktDto ?

Reply:
JsonTypeInfo.Id.NAME har default property "@type"

Reply:
Tror det henger igjen fra jeg eksperimenterte med å få polymorfisme til å fungere. Men ser jo at det der er allerede default verdiene, så fjerner begge to 👍